files/en-us/web/api/node/comparedocumentposition/index.html

```diff
@@ -16,66 +16,80 @@
 
 <p>The
     <code><strong>Node.compareDocumentPosition()</strong></code> method reports the
-    position of the given node relative to another node in any document — not just the
-    given node’s document.</p>
+    position of its argument node relative to the node on which it is called.</p>
 
-<p>The return value is a <a
-    href="https://en.wikipedia.org/wiki/Mask_(computing)">bitmask</a> of the following
-  values:</p>
+<h2 id="Syntax">Syntax</h2>
+
+<pre class="brush: js"><em>compareMask</em> = <em>node</em>.compareDocumentPosition(<em>otherNode</em>)
```

Comment:
```suggestion
<pre class="brush: js">compareDocumentPosition(otherNode)
```
Syntax blocks now lose all the extra styling and return value. That means you'll need to revisit the description that refers to the bits that are removed.

FWIW this all looks like a step in the right direction.

Reply:
The receiver is very relevant to understanding of this method's behavior, could we keep it as in the recently-modified [randomUUID](https://github.com/mdn/content/blob/2c5018d3cbc1c8880b303032eda6af38e8b1d24e/files/en-us/web/api/crypto/randomuuid/index.html#L24)?
```suggestion
<pre class="brush: js">node.compareDocumentPosition(otherNode)
```

Reply:
I do understand why you want to do this, but there is also a lot to be said for consistent documentation formatting:
- There was much discussion on the content of the syntax block, and pretty much it resolved down to the syntax section on functions being used as an indication of the parameters, rather than the thing that the item is called on or the return type
- It is possible to discuss the method behaviour without relying on the syntax block having that `node.` prefix. 

@Elchi3 Would you care to comment?

Reply:
Thank you. If it is indeed important that the receiver not be named in the syntax block, then I will make the changes (although I am curious about randomUUID being merged _with_ a named receiver just two days ago).

Reply:
Different reviewers catch different things. I would not have approved that change without making the comment that it does not match the guidelines. You will find a heap of docs that have not yet been migrated to the new format.

I figured I'd wait on @Elchi3 to comment on how important this is for this case. I expect that we will have the general rule for 99% of cases but that doesn't mean there won't be cases where someone says "really, let's leave this alone"

Reply:
The syntax section in this PR looks good to me (now). The docs on what we agreed are here, btw: https://developer.mozilla.org/en-US/docs/MDN/Structures/Syntax_sections